cypress/e2e/pages/homePage/index.cy.js

```diff
@@ -81,7 +229,46 @@ if (Cypress.env('SMOKE')) {
   });
 }
 
+const atiAnalyticsliteTestSuites = atiAnalyticsTestSuites
+  .filter(({ path }) => !path.startsWith('/persian/afghanistan'))
+  .map(testSuite => {
+    const excludedLiteTests = [
+      assertDropdownNavigationComponentView, // Dropdown navigation removed from all pages, as it requires JS
+      assertDropdownNavigationComponentClick, // Dropdown navigation removed from all pages, as it requires JS
+    ];
+
+    const liteSiteTests = testSuite.tests.filter(
+      test => !excludedLiteTests.includes(test),
+    );
+
+    // All lite enabled pages should have the Lite Site Summary component
+    liteSiteTests.push(assertLiteSiteSummaryComponentToMainSiteClick);
+
+    return {
+      ...testSuite,
+      path: getPathWithSuffix({ path: testSuite.path, suffix: '.lite' }),
+      applicationType: 'lite',
+      useReverb: false,
+      siteId: testSuite.service === 'magyarul' ? 134 : testSuite.siteId,
```

Comment:
Can we make sure we reinstate the tests on local + test for magyarul? It can just be an assertPageView (no components) because there's an issue with ATI that we want these tests to capture in future.